frontend/src/components/header/Header.js

```diff
@@ -1,56 +1,59 @@
-import React, {Component} from 'react';
-import {Grid} from '@material-ui/core';
-import logo from '../../assets/ethBuenosAires.png';
+import React, { Component } from 'react'
+import { Grid } from '@material-ui/core'
+import logo from '../../assets/ethBuenosAires.png'
+import logoLight from '../../assets/ethBuenosAires-light.svg'
 import { Link } from 'react-router-dom'
 
 import './header.scss'
 
-
-
 export default class HeaderWhite extends Component {
```

Comment:
Let's progressively add `'prop-types'` to our components since it doesn't take us lots of time and we get in exchange a better-documented component.

Example:

```javascript
import { Component } from 'react';
import PropTypes from 'prop-types';

export default class HeaderWhite extends Component {
...
}

HeaderWhite.propTypes = {
  white: PropTypes.string().isRequired,
  selected: PropTypes.bool().isRequired
}
```